backend/src/Validation.hs

```diff
@@ -113,8 +113,7 @@ validateStrongholds report
 
 validateReport :: GameReport -> Validation [ReportError] GameReport
 validateReport report =
-  report
```

Comment:
Our discussion today made me realize the fmap is entirely unnecessary :smile: 
Don't need to fmap the report into the Validation... it's already in there.